.openpublishing.redirection.json

```diff
@@ -1329,3176 +1329,4747 @@
+            "redirect_document_id": true
         },
         {
-            "source_path": "docs/csharp/programming-guide/concepts/linq/sample-xml-documents-linq-to-xml.md",
-            "redirect_url": "/dotnet/csharp/programming-guide/concepts/linq/sample-xml-file-typical-purchase-order-linq-to-xml-1"
+            "source_path": "docs/csharp/programming-guide/concepts/linq/functional-construction-linq-to-xml.md",
+            "redirect_url": "/dotnet/standard/linq/functional-construction",
+            "redirect_document_id": true
         },
         {
-            "source_path": "docs/csharp/programming-guide/concepts/linq/serializing-xml-trees.md",
-            "redirect_url": "/dotnet/csharp/programming-guide/concepts/linq/serializing-to-files-textwriters-and-xmlwriters"
+            "source_path": "docs/csharp/programming-guide/concepts/linq/functional-programming-vs-imperative-programming.md",
+            "redirect_url": "/dotnet/standard/linq/functional-vs-imperative-programming",
+            "redirect_document_id": true
         },
         {
-            "source_path": "docs/csharp/programming-guide/concepts/linq/tutorial-chaining-queries-together.md",
-            "redirect_url": "/dotnet/csharp/programming-guide/concepts/linq/deferred-execution-and-lazy-evaluation-in-linq-to-xml"
+            "source_path": "docs/csharp/programming-guide/concepts/linq/functional-transformation-of-xml.md",
+            "redirect_url": "/dotnet/standard/linq/functional-transformation-xml",
+            "redirect_document_id": true
         },
         {
-            "source_path": "docs/csharp/programming-guide/concepts/linq/tutorial-manipulating-content-in-a-wordprocessingml-document.md",
-            "redirect_url": "/dotnet/csharp/programming-guide/concepts/linq/shape-of-wordprocessingml-documents"
+            "source_path": "docs/csharp/programming-guide/concepts/linq/functional-vs-procedural-programming-linq-to-xml.md",
+            "redirect_url": "/dotnet/standard/linq/functional-vs-procedural-programming",
+            "redirect_document_id": true
         },
         {
-            "source_path": "docs/csharp/programming-guide/concepts/linq/working-with-xml-namespaces.md",
-            "redirect_url": "/dotnet/csharp/programming-guide/concepts/linq/namespaces-overview-linq-to-xml"
+            "source_path": "docs/csharp/programming-guide/concepts/linq/getting-started-linq-to-xml.md",
+            "redirect_url": "/dotnet/standard/linq/linq-xml-overview",
+            "redirect_document_id": false
         },
         {
-            "source_path": "docs/csharp/programming-guide/concepts/threading/how-to-use-a-thread-pool.md",
-            "redirect_url": "/dotnet/api/system.threading.threadpool.queueuserworkitem"
-        },
+            "source_path": "docs/csharp/programming-guide/concepts/linq/getting-started-with-linq.md",
+            "redirect_url": "/dotnet/csharp/programming-guide/concepts/linq/"
+        }
```

Comment:
```suggestion
        },
```

Reply:
Thanks @Youssef1313. I took care of the issues on my last commit.